test/e2e/common/setup-kubestellar.sh

```diff
@@ -65,6 +65,30 @@ make install-local-chart KUBE_CONTEXT=kind-kubeflex
 cd -
 echo "wds1 created."
 
+:
+: -------------------------------------------------------------------------
+: Run OCM transport controller executable
+:
+cd "${SRC_DIR}/../../.." ## go up to KubeStellar directory
+KUBESTELLAR_DIR="$(pwd)"
+## this is a temp solution - run as executble process. this should be replaced to run as pod, ideally using helm
+wget https://github.com/kubestellar/ocm-transport-plugin/archive/refs/tags/v0.1.0-rc1.tar.gz
+tar -xf v0.1.0-rc1.tar.gz
+rm -rf v0.1.0-rc1.tar.gz
+cd ocm-transport-plugin-0.1.0-rc1
+OCM_TRANSPORT_PLUGIN_DIR="$(pwd)"
+pwd
+echo "replace github.com/kubestellar/kubestellar => ${KUBESTELLAR_DIR}/" >> go.mod
+make build
+mv ./bin/ocm-transport-plugin ${KUBESTELLAR_DIR}/ocm-transport-plugin
+cd "${KUBESTELLAR_DIR}"
+pwd
+rm -rf ${OCM_TRANSPORT_PLUGIN_DIR}
```

Comment:
Doesn't this belong in cleanup?

Reply:
in the setup I left only the needed files on filesystem. only what is a must.
then the cleanup script removes the things that are a must - e.g., rm log file + kill the process + rm the executable file.
I can move this also to cleanup but this can be removed as soon as I untar the dir. no need to keep it.

Reply:
Yes, I see what is currently in the PR.

Note that sometimes things crash in the middle, so a later cleanup step is not reached. And doing `try ... finally ...` is hostile to anyone trying to debug what went wrong. So we have taken a different approach: a test starts with the cleanup. That would even involve removing directories that `tar xf` would later make.

Reply:
right. if that is not a big concern (which I understand it's not), let's leave as is and remove in the PR that would run a containerized transport controller.

Reply:
yes it is.  
you mean 0.1.0-rc2 right?
cause officially 0.1.0 still doesn’t exist.. we are still in the process of finalizing and testing 0.1.0.
so I should update otp to point to release 0.20.0-rc1 and create 0.1.0-rc2

Reply:
ks/ks now has release 0.20.0, OTP should depend on that now.
Yes, I think it would be best to make an OTP release (candidate) and make this PR use the pre-built stuff from that release.

Reply:
done

Reply:
Maybe there is a misunderstanding here. I still see `make build` above. I will continue the discussion in the thread that @ezrasilvera started.